cloudformation.properties

```diff
@@ -2,15 +2,21 @@ name=Cloudformation
 category=External Analysers
 descripton=Cloudformation rules for json and yaml, integrates cfn-nag reports.
 homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
-archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4
-publicVersions=2.0.5
+archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4,2.0.5
+publicVersions=2.0.7
 
 defaults.mavenGroupId=com.hack23.sonar
 defaults.mavenArtifactId=sonar-cloudformation-plugin
 
+2.0.7.description=Bugfixes, updated rule descriptions and nist 800-53 tags.
+2.0.7.date=2020-10-31
+2.0.7.sqVersions=[7.9,LATEST]
+2.0.7.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.0.7
```

Comment:
Not required, but it would be nice in your changelog to link the 2.0.6 mention to its changelog - to give users an easy & transparent way to understand _all_ the changes in the version.